zapisy/apps/schedule/views.py

```diff
@@ -263,7 +263,10 @@ def change_interested(request, event_id):
 
 @login_required
 def get_terms(request, year, month, day):
-    date = datetime.date(int(year), int(month), int(day))
+    try:
+        date = datetime.date(int(year), int(month), int(day))
+    except ValueError:
+        raise Http404
```

Comment:
Rzadko takie rzeczy do tej pory robiliśmy (zwłaszcza że ich efekt widać tylko w Rollbarze), ale na pewno nie zaszkodzi.
```suggestion
        raise Http404("URL contains invalid date.")
```